dotnet/targets/Xamarin.Shared.Sdk.targets

```diff
@@ -1233,6 +1234,25 @@
 		</ItemGroup>
 	</Target>
 
+	<!-- When we're building a universal app, we need to collect the symbols from each inner build -->
+	<Target Name="_CollectRidSpecificSymbolLists" Inputs="@(_RidSpecificSymbolsListPath)" Outputs="$(_MtouchSymbolsList)">
+		<ReadLinesFromFile File="%(_RidSpecificSymbolsListPath.Identity)">
+			<Output TaskParameter="Lines" ItemName="_AllExecutableSymbols" />
+		</ReadLinesFromFile>
+		<WriteLinesToFile File="$(_MtouchSymbolsList)" Lines="@(_AllExecutableSymbols->DistinctWithCase())" Overwrite="true" />
+	</Target>
+
+	<!-- When we're building a universal app, we need to collect the list of user frameworks we need to run dsymutil on -->
+	<Target Name="_CollectRidSpecificUserFrameworksWithoutDebugSymbols">
+		<ReadItemsFromFile
+			SessionId="$(BuildSessionId)"
```

Comment:
Minor comment here. When building from Windows, if the real file exists there, we don't need to add the SessionId since we don't need to remote the execution of this task.

Reply:
Ah, that's good to know that I can just skip the SessionId property to execute on Windows instead of remoting.

In this case this is somewhat urgent to get in (in time for the RC), and any changes would require a lot of re-testing, so I think I'll merge this PR and then maybe look at this later.